caffe2/CMakeLists.txt

```diff
@@ -1076,6 +1076,14 @@ if(USE_XPU)
         "-Wl,--whole-archive,\"$<TARGET_FILE:torch_xpu_ops>\" -Wl,--no-whole-archive")
     endif()
   endif()
+
+  # Set exported torch-xpu-ops header to torch
+  set(INCLUDE_DIR_FOR_TORCH "${CMAKE_BINARY_DIR}/aten/src/ATen/xpu" )
+  file(MAKE_DIRECTORY ${INCLUDE_DIR_FOR_TORCH})
+  file(GLOB xpu_h "${TORCH_ROOT}/third_party/torch-xpu-ops/src/aten/*.h")
```

Comment:
We should not use hard-coded dir, but use a cmake variable outputed by `add_directory(torch-xpu-ops)`. Since the code will couple with torch-xpu-ops implementation. The variable could align with third_party packages, like MKLDNN_INCLUDE_DIR.